packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -43,6 +43,7 @@
 			</SplitterPanel>
 			<SplitterPanel
 				class="project-page"
+				style="z-index: 2; /** temporary styling to make drilldown contents above workflow nodes */"
```

Comment:
Let's do the proper fix instead of a temporary one

Reply:
jamie made a similar change - merged in main atm